src/promptflow-devkit/promptflow/_sdk/_service/static/assets/style.css

```diff
@@ -1 +1 @@
dg-color-scheme: dark;--rdg-color: #ddd;--rdg-border-color: #444;--rdg-summary-border-color: #555;--rdg-background-color: hsl(0deg 0% 13%);--rdg-header-background-color: hsl(0deg 0% 10.5%);--rdg-header-draggable-background-color: hsl(0deg 0% 17.5%);--rdg-row-hover-background-color: hsl(0deg 0% 9%);--rdg-row-selected-background-color: hsl(207deg 76% 42%);--rdg-row-selected-hover-background-color: hsl(207deg 76% 38%);--rdg-checkbox-color: hsl(207deg 100% 79%);--rdg-checkbox-focus-color: hsl(207deg 100% 89%);--rdg-checkbox-disabled-border-color: #000;--rdg-checkbox-disabled-background-color: #333}.r104f42s7-0-0-beta-39.rdg-light{--rdg-color-scheme: light}@media (prefers-color-scheme: dark){.r104f42s7-0-0-beta-39:not(.rdg-light){--rdg-color: #ddd;--rdg-border-color: #444;--rdg-summary-border-color: #555;--rdg-background-color: hsl(0deg 0% 13%);--rdg-header-background-color: hsl(0deg 0% 10.5%);--rdg-header-draggable-background-color: hsl(0deg 0% 17.5%);--rdg-row-hover-background-color: hsl(0deg 0% 9%);--rdg-row-selected-background-color: hsl(207deg 76% 42%);--rdg-row-selected-hover-background-color: hsl(207deg 76% 38%);--rdg-checkbox-color: hsl(207deg 100% 79%);--rdg-checkbox-focus-color: hsl(207deg 100% 89%);--rdg-checkbox-disabled-border-color: #000;--rdg-checkbox-disabled-background-color: #333}}}}@layer rdg.Root{.v7ly7s7-0-0-beta-39{-webkit-user-select:none;user-select:none}.v7ly7s7-0-0-beta-39 .r1otpg647-0-0-beta-39{cursor:move}}@layer rdg.FocusSink{.fc4f4zb7-0-0-beta-39{grid-column:1/-1;pointer-events:none;z-index:1}}@layer rdg.FocusSink{.fq51q037-0-0-beta-39{z-index:3}}@layer rdg.SummaryCell{.s1n3hxke7-0-0-beta-39{inset-block-start:var(--rdg-summary-row-top);inset-block-end:var(--rdg-summary-row-bottom)}}@layer rdg.SummaryRow{.snfqesz7-0-0-beta-39{line-height:var(--rdg-summary-row-height)}.snfqesz7-0-0-beta-39>.c1wupbe7-0-0-beta-39{position:sticky}}@layer rdg.SummaryRow{.t1jijrjz7-0-0-beta-39>.c1wupbe7-0-0-beta-39{z-index:2}.t1jijrjz7-0-0-beta-39>.cd0kgiy7-0-0-beta-39{z-index:3}}@layer rdg.SummaryRow{.t14bmecc7-0-0-beta-39>.c1wupbe7-0-0-beta-39{border-block-end:2px solid var(--rdg-summary-border-color)}}@layer rdg.SummaryRow{.b1odhhml7-0-0-beta-39>.c1wupbe7-0-0-beta-39{border-block-start:2px solid var(--rdg-summary-border-color)}}@layer rdg.GroupedRow{.gyxx7e97-0-0-beta-39:not([aria-selected=true]){background-color:var(--rdg-header-background-color)}.gyxx7e97-0-0-beta-39>.c1wupbe7-0-0-beta-39:not(:last-child):not(.c1730fa47-0-0-beta-39){border-inline-end:none}}@layer rdg.TextEditor{.tlmcuo07-0-0-beta-39{-webkit-appearance:none;-moz-appearance:none;appearance:none;box-sizing:border-box;inline-size:100%;block-size:100%;padding-block:0;padding-inline:6px;border:2px solid #ccc;vertical-align:top;color:var(--rdg-color);background-color:var(--rdg-background-color);font-family:inherit;font-size:var(--rdg-font-size)}.tlmcuo07-0-0-beta-39:focus{border-color:var(--rdg-selection-color);outline:none}.tlmcuo07-0-0-beta-39::placeholder{color:#999;opacity:1}}
```

Comment:
may add some PR description on what this change brings.

Reply:
Added in the description